js/screens/SearchResult/SearchResultScreen.js

```diff
@@ -34,45 +38,136 @@ class SearchResultScreen extends Component {
     };
   };
 
+  getChipLabels() {
```

Comment:
you can use the arrow function and not do the binding on line 25.

Reply:
done.